crates/project/src/lib.rs

```diff
@@ -533,6 +532,17 @@ where
         let mut results = Vec::new();
 
         for test in tests {
+            let path = format!("{}{}", test.module, test.name);
+
+            match match_tests.clone() {
+                None => {}
+                Some(search_str) => {
+                    if !path.to_string().contains(&search_str) {
+                        continue;
+                    }
+                }
+            }
```

Comment:
this could be an if let

```rust
if let Some(search_str) = match_tests {
    if !path.to_string().contains(&search_str) {
        continue;
    }
}
```

or even

```rust
if matches!(match_tests, Some(search_str) if !path.to_string().contains(search_str) ) {
    continue;
}
```